guides/common/assembly_installing-capsule-server.adoc

```diff
@@ -10,18 +10,15 @@ For more information, see {InstallingSmartProxyDocURL}preparing-environment-for-
 ifdef::satellite,katello[]
 // Registering {SmartProxyServer} to {ProjectServer}
 include::modules/proc_registering-capsule-to-satellite-server.adoc[leveloffset=+1]
+endif::[]
 
 ifdef::satellite[]
 // Attaching the Satellite Infrastructure Subscription
 include::modules/proc_attaching-satellite-infrastructure-subscription.adoc[leveloffset=+1]
 endif::[]
 
+ifdef::foreman-el,foreman-deb,satellite[]
```

Comment:
I'm unsure about Katello and Orcharhino here. I think it should be done, but I wasn't quite sure.

Reply:
@maximiliankolb ?

Reply:
Now that I read through it again, I see it was previously done for Katello. I was confused by the registering part. In theory it's possible to sync content to Katello, register the Proxy to Katello and get the content that way. I just don't know if that's a desired (primary) workflow.

Reply:
This should not be visible for orcharhino builds. Actually, I will put it on my todo to simply make the whole install/upgrade guide blank for orcharhino, as we use a different installation and upgrade process. One big ifndef in the master files should be easier to maintain for now.